tools/ExtensionsMetadataGenerator/src/ExtensionsMetadataGenerator/runtimeassemblies.txt

```diff
@@ -1,17 +1,24 @@
+Microsoft.Azure.KeyVault.Core.dll
+Microsoft.OData.Core.dll
+Microsoft.OData.Edm.dll
+Microsoft.Spatial.dll
+Microsoft.Azure.Storage.Common.dll
+Microsoft.Azure.DocumentDB.Core.dll
+Microsoft.Azure.Cosmos.Table.dll
+Microsoft.Azure.Storage.Blob.dll
+Microsoft.Azure.Storage.File.dll
+Microsoft.Azure.Storage.Queue.dll
```

Comment:
This makes me nervous -- a lot of people use newer versions of these Storage assemblies and we can't just remove them if they're there (same with App Insights below).

Reply:
the reason ApplicationIsights is added here is  - it is listed in runtimeassemblies.json - https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/runtimeassemblies.json#L53-L56 and we are enforcing the version loaded by host and not what users brings in. 

Please do correct me if I misunderstood. If these assemblies should not be cleaned up then we should remove them from runtimeassemblies.json ?